environment/commands/clean.js

```diff
@@ -17,13 +17,14 @@ class CleanCommand {
             });
         });
         try {
-            execSync('python3 ~/.rainbowup/nearup/nearup stop');
+            execSync('python3 ~/.rainbowup/nearup/main.py stop');
```

Comment:
This was incorrect,  `nearup` will update and use `~/.nearup`. `main.py` use `~/.rainbowup/nearup/main.py`